pkg/virt-controller/watch/snapshot/restore.go

```diff
@@ -32,17 +32,21 @@ import (
 	storagev1 "k8s.io/api/storage/v1"
 	"k8s.io/apimachinery/pkg/api/equality"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/sets"
 
 	kubevirtv1 "kubevirt.io/api/core/v1"
 	snapshotv1 "kubevirt.io/api/snapshot/v1alpha1"
 	"kubevirt.io/client-go/log"
 
+	"kubevirt.io/containerized-data-importer-api/pkg/apis/core/v1beta1"
+
 	"kubevirt.io/kubevirt/pkg/controller"
+	watchutil "kubevirt.io/kubevirt/pkg/virt-controller/watch/util"
```

Comment:
Since cleaning up imports, it's best to have three blocks/sections first, system stuff, then non kubevirt.io, then kubevirt.io

Reply:
so better 4 sections: (1)system; (2)non-k8s.io/kubevirt.io; (3)k8s.io; (4)kubevirt.io. ack?